js/ascendex.js

```diff
@@ -1285,11 +1282,16 @@ module.exports = class ascendex extends Exchange {
             // 'page': 1,
             // 'pageSize': 100,
         };
+        let market = undefined;
         if (symbol !== undefined) {
             market = this.market (symbol);
             request['symbol'] = market['id'];
         }
-        const method = this.safeValue (options, 'method', 'v1PrivateAccountGroupGetOrderHist');
+        const [ methodType, query ] = this.handleMarketTypeAndParams ('fetchOpenOrders', market, params);
```

Comment:
Should be `fetchClosedOrders` here.

Reply:
On line 1290 I'm not sure how to fix this error that happens if I switch 'fetchOpenOrders' to 'fetchClosedOrders':
[NotSupported] ascendex [object Object] does not have a value in mapping